history/views.py

```diff
@@ -564,7 +567,8 @@ def profit_view(request):
     # get data
     data = {}
     for t in Trade.objects.filter(symbol=symbol, status='fill').order_by('-created_on').all():
-        date = datetime.datetime.strftime(t.created_on-datetime.timedelta(hours=7), '%Y-%m-%d')
+        mst_dt = timezone.localtime(t.created_on, pytz.timezone('MST'))
+        date = datetime.datetime.strftime(mst_dt, '%Y-%m-%d')
```

Comment:
IIRC, django-chartit requires the data to be available within the queryset, so a python property will likely not work.

data is posted @ https://github.com/owocki/pytrader/issues/2#issuecomment-205048104 not asking that you make the change, just telling you about this because now that you've made a PR you should at least have access to the data so you can trade :)